frontend/packages/dev-console/integration-tests/cypress.json

```diff
@@ -23,10 +23,10 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "env": {
-    "TAGS": "@smoke and not @manual"
+    "TAGS": "@smoke or @regression and not @manual"
   },
   "retries": {
     "runMode": 1,
     "openMode": 0
   }
-}
+}
```

Comment:
Add new line to the existing file

Reply:
Please run "yarn run prettier" command for every pr, to avoid lint issues

Reply:
> Please run "yarn run lint" command for every pr, to avoid lint issues

Reply:
sure, thanks. will do

Reply:
done